### PR TITLE
Autoplay next

### PR DESCRIPTION
**Pull request by jessopb**
WIP only because currently choosing the first in related can result in a loop between 2 or 3 videos.

### CODE REVIEW

File: src/ui/component/viewers/videoViewer/view.jsx [hunk 1]
```diff
@@ -14,16 +14,58 @@ const VIDEO_JS_OPTIONS = {
...
+    }
+    // cleanup function:
+    return () => {
+      if (currentVideo) {
+        currentVideo.removeEventListener('ended', doEnded);
+        currentVideo.removeEventListener('pause', doPause);
+        currentVideo.removeEventListener('volumechange', doVolume);
+      }
+    };
+  }, []);
```
Comment (seanyesmunt) [inline src/ui/component/viewers/videoViewer/view.jsx]: Does the videoviewer always leave the page then get re-rendered again?

**Comment by seanyesmunt** [comment]
This looks good. Nice work so far

**Comment by jessopb** [comment]
Now we (mostly) only play unvisited links. However, we need to modify the doSearch to take some smarter options like mediaType.

**Comment by seanyesmunt** [comment]
This looks fine. Closing this since it's contained in #2861

**Pull request closed without merging**